ios/RNSScreenStack.mm

```diff
@@ -753,6 +753,8 @@ - (BOOL)gestureRecognizerShouldBegin:(UIGestureRecognizer *)gestureRecognizer
       // of `UISemanticContentAttributeForceLeftToRight`, so we just check if it is RTL or not
       BOOL isCorrectEdge = (_controller.view.semanticContentAttribute == UISemanticContentAttributeForceRightToLeft &&
                             ((RNSScreenEdgeGestureRecognizer *)gestureRecognizer).edges == UIRectEdgeRight) ||
+          (topScreen.stackAnimation == RNSScreenStackAnimationSlideFromLeft &&
+           ((RNSScreenEdgeGestureRecognizer *)gestureRecognizer).edges == UIRectEdgeRight) ||
```

Comment:
Why is this needed? I can't see any issues on RTL mode with and without this case 🤔

Reply:
@tboba this is needed to activate the gesture from the right side if "slide_from_left" transition is used.

But I guess this condition needs to be slightly modified to support gesture from left side if RTL is active, right?

Reply:
Yeah, probably it needs to be modified, since on RTL, the default behavior to dismiss the screen is to swipe from the right side, but unfortunately I can't make the reversed slide work 🤔 For some reason, I can still dismiss that screen from the left side. Does it work for you?

https://github.com/software-mansion/react-native-screens/assets/23281839/4ded1730-ce9f-4f11-9430-377be74b928e

Reply:
> Yeah, probably it needs to be modified, since on RTL, the default behavior to dismiss the screen is to swipe from the right side, but unfortunately I can't make the reversed slide work 🤔

@tboba I think in normal mode (not RTL) if "slide_from_left" is specified, then the gesture to dismiss should be "right_to_left".

If RTL is active, then we reverse everything - so "slide_from_left" will actually move screen from right to left (as default iOS transition) and the gesture for dismissing should be from left corner (saying shorter - if RTL + "slide_from_left" is specified, then it should work as usual/default iOS transition, because it's kind of double inversion). 

Is my understanding correct?

On your video - did you specify `customAnimationOnSwipe` property? Because without this property you're right -  the gesture to close should start from left corner and should have default transition (even if custom transition was specified).

Reply:
> On your video - did you specify customAnimationOnSwipe property?

I'm asking because this condition can be executed only if `customAnimationOnSwipe` specified.

```objc
  // Now we're dealing with RNSScreenEdgeGestureRecognizer (or _UIParallaxTransitionPanGestureRecognizer)
  if (topScreen.customAnimationOnSwipe && [RNSScreenStackAnimator isCustomAnimation:topScreen.stackAnimation]) {
```

Reply:
@tboba I re-worked this condition according to my comment - feel free to test it 🙌 

I also added named variables to simplify condition and made it more readable 😊

Reply:
Great, I'll try to test them today! Thanks 🙌

> If RTL is active, then we reverse everything - so "slide_from_left" will actually move screen from right to left (as default iOS transition) and the gesture for dismissing should be from left corner (saying shorter - if RTL + "slide_from_left" is specified, then it should work as usual/default iOS transition, because it's kind of double inversion).

Yep, you're right - currently, the default animation comes from the right side to the left and is rotated when user is in RTL mode - I think in `slide_to_left` transition it should be opposite!
I also forgot about the `customAnimationOnSwipe` prop 😅 I'll add it today 👍